package.json

```diff
@@ -48,15 +63,22 @@
     "react-redux": "4.4.8",
     "react-router": "3",
     "react-scroll": "1.5",
+    "react-test-renderer": "^15.5.4",
     "react-transition-group": "^1.1.2",
     "recompose": "^0.21.2",
     "redux": "^3.5.2",
-    "redux-thunk": "^2.1.0"
+    "redux-thunk": "^2.1.0",
+    "sinon": "^3.2.1",
+    "skin-deep": "^1.0.0"
   },
   "dependencies": {
     "babel-plugin-dynamic-import-node": "^1.2.0",
+    "blob-polyfill": "^2.0.20171115",
+    "choma": "^1.1.0",
     "downshift": "^1.22.5",
     "@department-of-veterans-affairs/react-jsonschema-form": "^1.0.0",
+    "eslint-plugin-import": "^2.7.0",
```

Comment:
Wondering why these two eslint deps are not in devDeps?

Reply:
Good question, short answer is I've been transferring a lot of the set up from the Vets.gov codebase and that's how these packages were installed, as regular dependencies. Looking into them, I think it's possible to just have them installed as devDependencies, so I'll change that for now and we can always switch back if for some reason an issue arises with that in the future.

Reply:
👍